.custom-gcl.yml

```
@@ -0,0 +1,4 @@
+version: v1.57.0
```

```suggestion
version: v1.62.0
```

To be consistent with https://github.com/google/go-github/blob/27bd17565a213b72d232e8e1ea820881f791f1a7/script/lint.sh#L8